main_doc/styles.scss

```diff
@@ -0,0 +1,453 @@
+                width: 25%;
+            }
+
+            h2 {
+                color: #FFF;
+                text-align: right;
+                font-family: Sen, sans-serif;
+                font-size: 144px;
+                font-weight: 800;
+                line-height: 96%;
+                text-transform: uppercase;
+                position: relative;
+                z-index: 2;
+            }
+        }
+        .work-img {
+            height: 334px;
+            position: relative;
+            z-index: 2;
+
+            .work-different-img {
+                display: flex;
+                justify-content: space-between;
+                align-items: flex-end;
+                height: 100%;
+            }
+        }
+        .work-button {
+            position: absolute;
+            bottom: 5%;
+            right: calc(50% - 96px);
+            z-index: 2;
+        }
+    }
+}
+.baker {
+    height: 840px;
+    position: relative;
+    width: 100%;
+     
+    .bg-body-baker {
+        background-color: $main-color;
+        width: 67%;
+        height: 840px;
+        margin: 0;
+    }  
+    .about-us-baker {
+        position: absolute;
+        top: 0;
+        display: flex;
+        justify-content: flex-start;
+        height: 840px;
+        align-items: center;
+
+        .baker-information {
+            width: 40%;
+            display: flex;
+            flex-direction: column;
+            justify-content: space-around;
+            height: 840px;
+            margin-right: 5%;
+        
+            h2 {
+                color: #FFF;
+                text-align: left;
+                font-family: Sen, sans-serif;
+                font-size: 144px;
+                font-weight: 800;
+                line-height: 96%;
+                text-transform: uppercase;
+                margin: 0;
+            }
+
+            p {
+                color: #FFF;
+                font-size: 18px;
+                font-weight: 600;
+                line-height: 26px;
+                text-align: justify;
+            }
+        }
+    }
+}
+
+.servises {
+    height: 840px;
+
+    .servises-content {
+        height: 840px;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        align-content: center;
+        flex-wrap: wrap;
+        
+        h2 {
+            color: $main-color;
+            font-family: Sen, sans-serif;
+            font-size: 144px;
+            font-weight: 800;
+            line-height: 96%;
+            text-transform: uppercase;
+            text-align: center;
+        }
+
+        .servises-type {
+            width: 80%;
+            display: flex;
+            justify-content: space-evenly;
+
+            .bakery-card-product {
+                border-color: #FFD8AB;
+                border-radius: 8%;
+                
+                .bakery-name-product {
+                    color: $main-color;
+                    font-size: 16px;

```

Comment:
in `font-sizes` better to use values in `em`

Reply:
it's not a mistake, but you will have bigger flexibility in future if you will use `em` in the whole project